src/main/java/calculator/string/token/IntTokens.java

```diff
@@ -0,0 +1,61 @@
+package calculator.string.token;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class IntTokens {
+    private final List<Integer> tokens;
+
+    private IntTokens(final String[] tokens) {
+        validate(tokens);
+
+        this.tokens = parseInt(tokens);
+        validIsAllPositive();
+    }
+
+    private void validate(String[] tokens) {
+        if (tokens == null || tokens.length == 0) {
+            throw new IllegalArgumentException("String token is null");
+        }
+    }
+
+    public static IntTokens init(final String[] tokens) {
+        return new IntTokens(tokens);
+    }
+
+    public int calculate() {
+        return tokens.stream()
+                .mapToInt(Integer::intValue)
+                .sum();
+    }
+
+    private List<Integer> parseInt(final String[] tokens) {
```

Comment:
개인적으로 parseInt()는 변수 하나만 변환하는 느낌의 메소드 이름인 것 같은데, (Integer에도 있는 메소드라)
```suggestion
    private List<Integer> parseInts(final String[] tokens) {
```
명확하게 바꿔보는건 어떨까요?

Reply:
넵 ㅋㅋ 그게 낫겠네요